client/src/components/form/Checkboxes/Checkboxes.jsx

```diff
@@ -10,6 +10,12 @@ const Checkboxes = ({
   maxCanSelect,
   label,
 }) => {
+  useEffect(() => {
+    if (initialSelectedIndices !== undefined) {
```

Comment:
Is there any case where these could be `undefined`?

Reply:
Yes! Usually it will be undefined
This is just if say you wanted a list of Apples, Oranges, and Bananas but Apples were checked by default